### ISSUE TITLE
unable to install geocat in windows

### ISSUE THREAD

**Issue by CHIRANJIT007-HUE** [NCAR/geocat-comp#131]
Hey everyone,
                        I am unable to install the GeoCAT using 'conda install -c ncar geocat-comp' this conda run. Any help would be appreciated.

**Comment by erogluorhan** [comment]
Hi @CHIRANJIT007-HUE ,

Sorry for the inconvenience you are having!

Unfortunately, we do not have a Windows support for GeoCAT-comp; however, the good news is that we are currently investigating it and will hopefully take actions for providing that support.

Please do not hesitate to ask further questions/requests.

Orhan - GeoCAT

**Comment by cyschneck** [comment]
Is there still an issue to download on Windows?

I attempted to replicate the issue with the [following steps](https://github.com/conda-forge/geocat-comp-feedstock) on a new conda env

Create a new environment and activate
```
conda create --name geocat_131 python=3.10
conda activate geocat_131
```
Add `conda-forge` to channels
```
conda config --add channels conda-forge
conda config --set channel_priority strict
```
Install `geocat-comp`
```
conda install geocat-comp
```
After a few minutes to install, it appears to be running on a Windows env with the most recent version `v2023.10.1`
```
Python 3.10.13 | packaged by conda-forge | (main, Oct 26 2023, 18:01:37) [MSC v.1935 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import geocat.comp as gc
>>> gc.__version__
'2023.10.1'
```

**Comment by cyschneck** [comment]
Will be closed as a part of https://github.com/NCAR/geocat-comp/issues/463 with the fixes to Windows tests

**Comment by cyschneck** [comment]
Closed as part of Issue #463 (PR #516 ) with `windows-latest` tests
